src/control/server/ctl_storage_rpc.go

```diff
@@ -170,10 +171,31 @@ func (c *ControlService) scanScm(ctx context.Context, req *ctlpb.ScanScmReq) (*c
 	return newScanScmResp(c.getScmUsage(ssr))
 }
 
+// Check if a PCI is a VMD address
+func isVmdAddress(addr string) bool {
```

Comment:
Please use the existing function PCIAddress.IsVMDBackingAddress() in lib/hardware/pci.go for this, you might have to create a PCIAddress type from the string first.